packages/vertexai/src/errors.ts

```diff
@@ -1,63 +0,0 @@
-/**
- * @license
- * Copyright 2024 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { ErrorFactory, ErrorMap } from '@firebase/util';
-import { GenerateContentResponse } from './types';
-
-export const enum VertexError {
-  FETCH_ERROR = 'fetch-error',
-  INVALID_CONTENT = 'invalid-content',
-  NO_API_KEY = 'no-api-key',
-  NO_MODEL = 'no-model',
-  NO_PROJECT_ID = 'no-project-id',
-  PARSE_FAILED = 'parse-failed',
-  RESPONSE_ERROR = 'response-error'
-}
-
-const ERRORS: ErrorMap<VertexError> = {
-  [VertexError.FETCH_ERROR]: `Error fetching from {$url}: {$message}`,
-  [VertexError.INVALID_CONTENT]: `Content formatting error: {$message}`,
-  [VertexError.NO_API_KEY]:
-    `The "apiKey" field is empty in the local Firebase config. Firebase VertexAI requires this field to` +
```

Comment:
Let's make sure all these old error message strings are preserved in the new code - they now need to be added wherever new VertexError() is called.